### ISSUE TITLE
Make new release notifications less annoying

### ISSUE THREAD

**Issue by amot-dev** [amot-dev/mototwist#128]
Perhaps a new db table could store the last release notified about (and maybe the time?). Could also only notify admin users, and not regular/logged off users.

**Comment by amot-dev** [comment]
Should also ideally cache latest version. Don't need to be checking that on each load. Come to think of it, redis should be able to easily handle both of these tasks.

**Comment by amot-dev** [comment]
Re-opening to test a new potential issue-closing flow.
